tests/console/verify_systemd_timesync.pm

```diff
@@ -0,0 +1,59 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify that systemd timer is used for time synchronization.
+#          - Check if yast-timer configuration file exists and contains expected values.
+#          - Check configured time synchronization server address.
+#          - Check the message logs for "One time synchronization" occurrence.
+# Maintainer: Sofia Syrianidou <ssyrianidou@suse.com>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use testapi;
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+sub run {
+    my $test_data = get_test_suite_data();
+    select_console 'root-console';
+
+    record_info("Check configuration", "Check if file /etc/systemd/system/yast-timesync.timer exists and has expected configuration.");
+    assert_script_run('ls /etc/systemd/system/yast-timesync.timer');
+    my $conf_file = script_output("cat /etc/systemd/system/yast-timesync.timer");
+    $conf_file =~ /OnUnitActiveSec=(?<interval>(\S+))min/;
+    my $expected_interval = $test_data->{profile}->{'ntp-client'}->{ntp_sync};
+    assert_equals($expected_interval, $+{interval}, "The interval in yast-timesync configuration file is not the expected one.");
+
+    record_info("Check server", "Check if the configured time synchronization server is the expected one.");
+    my $expected_server = $test_data->{profile}->{'ntp-client'}->{ntp_servers}->{ntp_server}->{address};
+    my $chrony_conf     = script_output("cat /etc/chrony.conf");
+    $chrony_conf =~ /(\R|^)pool\s(?<server>(\S+))/;
+    assert_equals($expected_server, $+{server});
```

Comment:
In case of multiple server addresses, only the first one will be taken under consideration.

Reply:
I guess it is ok for this test and probably not needed a comment as the regex does not have `g` modifier `/regex/g` so only one match is expected. Otherwise it would need a loop.